### PR TITLE
Add Screenshots tab and audio capture

### PR DESCRIPTION
**Pull request by aliciakrebs**
### Description
Add Screenshots tab and audio capture

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by aoleszyn — APPROVED** [review]

File: docs/test-results/viewing-test-results.md [hunk 1]
```diff
@@ -83,10 +84,11 @@ Your test assets, including videos, screenshots, and logs, are retained for 30 d
 1. In Sauce Labs, in the left panel, click **AUTOMATED** or **LIVE** and then click **Test Results**.
```
Comment (lysannep) [inline docs/test-results/viewing-test-results.md:84]:
```suggestion
1. In Sauce Labs, in the left panel, click **Automated** or **Live** and then click **Test Results**.
```

File: docs/test-results/viewing-test-results.md [hunk 2]
```diff
@@ -83,10 +84,11 @@ Your test assets, including videos, screenshots, and logs, are retained for 30 d
...
 3. On the **Test Details** page, you can do the following:
-  * Click the play button to view screenshots and video of the test. Audio is not recorded.
   * Click the **COMMANDS** tab to see the API commands that were issued during the test. The **COMMANDS** tab also includes controls for filtering the commands displayed and playing back the commands.
+  * Click the **METADATA** tab to see the metadata attached to your test. Some of the test metadata, such as Passed, can be updated via the [Sauce REST API](/dev/api). 
+  * Click the **Video** tab to view video of the test. You can download the video from this tab. Audio is available for real devices only. Additional configurations are required to record audio with your tests. For more information, see [Audio Capture](/mobile-apps/features/audio-capture/).
+  * Click the **Screenshots** tab to view screenshots. You can download the screenshots from this tab, and view them in full screen.
   * Click the **LOGS** tab to see the logs generated by your test. The logs you can view are determined by the type of test you ran. For example, web app tests will include a Selenium log, while mobile app tests will contain an Appium log.
```
Comment (lysannep) [inline docs/test-results/viewing-test-results.md:91]:
```suggestion
  * Click the **Logs** tab to see the logs generated by your test. The logs you can view are determined by the type of test you ran. For example, web app tests will include a Selenium log, while mobile app tests will contain an Appium log.
```

File: docs/test-results/viewing-test-results.md [hunk 3]
```diff
@@ -83,10 +84,11 @@ Your test assets, including videos, screenshots, and logs, are retained for 30 d
 1. In Sauce Labs, in the left panel, click **AUTOMATED** or **LIVE** and then click **Test Results**.
 2. Click the test whose results you want to view.
 3. On the **Test Details** page, you can do the following:
-  * Click the play button to view screenshots and video of the test. Audio is not recorded.
   * Click the **COMMANDS** tab to see the API commands that were issued during the test. The **COMMANDS** tab also includes controls for filtering the commands displayed and playing back the commands.
```
Comment (lysannep) [inline docs/test-results/viewing-test-results.md:87]:
```suggestion
  * Click the **Commands** tab to see the API commands that were issued during the test. The **Commands** tab also includes controls for filtering the commands displayed and playing back the commands.
```

File: docs/test-results/viewing-test-results.md [hunk 4]
```diff
@@ -44,7 +45,7 @@ To view automated test results:
 
 2. On the **Test Results** page, click the **Virtual Cloud** or **Real Devices** radio button to view the relevant device results.
```
Comment (lysannep) [inline docs/test-results/viewing-test-results.md:47]:
```suggestion
```

Procedure steps should have single-line spacing (current output shown below). 

Screen Shot 2022-10-06 at 3 22 23 PM

File: docs/test-results/viewing-test-results.md (hunk 3 above)
Comment (lysannep) [inline docs/test-results/viewing-test-results.md:87]: The sentences for each bulleted point could also be updated so the focus is on what the value is for the user. For example: "To see the API commands that were issued during the test, click the **Commands** tab. (...)

**Pull request merged**